common/jsonquery/converters.go

```diff
@@ -66,3 +66,14 @@ func (convertor) ObjectToMap(node *ajson.Node) (map[string]any, error) {
 
 	return result, nil
 }
+
+// NodeToArray treats current node as an array of objects.
+// Sometimes JSON response from the server right of the bet is an array. This method performs the necessary conversion.
+func (convertor) NodeToArray(node *ajson.Node) ([]*ajson.Node, error) {
```

Comment:
is this wrapping necessary? we can just use node.GetArray() from the caller without this wrapper.

Reply:
It converts library returned error into `jsonquery.ErrNotArray`.
Overal most errors are either replaced or enhanced from ajson library. So the only benefit is that we can do error type checking.

Let me know if this should be removed. I will then restack with graphite.